cmd/simulator/full.go

```diff
@@ -16,6 +16,7 @@ import (
 	"github.com/ethereum/go-ethereum/rpc"
 	"github.com/smartcontractkit/libocr/commontypes"
 	"github.com/smartcontractkit/libocr/offchainreporting2/types"
+	"github.com/smartcontractkit/ocr2keepers/pkg/chain/client"
 
 	"github.com/smartcontractkit/ocr2keepers/internal/keepers"
 	"github.com/smartcontractkit/ocr2keepers/pkg/chain"
```

Comment:
can we add the import here? this way we keep the imports sorted

Reply:
Thanks for this feedback dude! What import settings do you use in your IDE?

Reply:
do we have this documented somewhere? Would be good to have as new devs join the team